CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGELOG
 
+### 2.0.x
+
+**SDK**
+- *Feature:* Added public interface to create a mini app using mini app id `MiniApp#create(appId:completionHandler:messageInterface)`
```

Comment:
The pre-fixes should be bold instead of italic: `**Feature:**`. Sorry, we changed the style since the last releases.

Reply:
Done